### PR TITLE
DCJ-638: Remove old approved users API

### PR DESCRIPTION
**Pull request by rushtong**
### Addresses
https://broadworkbench.atlassian.net/browse/DCJ-638

### Summary
Minor cleanup of the unused, original download dataset approved users endpoint

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/java/org/broadinstitute/consent/http/resources/DatasetResource.java [hunk 1]
```diff
@@ -575,21 +571,21 @@ public Response syncDataUseTranslation(@Auth AuthUser authUser, @PathParam("id")
...
+//      @PathParam("datasetId") Integer datasetId) {
+//    try {
+//      String content = darService.getDatasetApprovedUsersContent(authUser, datasetId);
+//      return Response.ok(content)
+//          .header(HttpHeaders.CONTENT_DISPOSITION, "attachment; filename=DatasetApprovedUsers.tsv")
+//          .build();
+//    } catch (Exception e) {
+//      return createExceptionResponse(e);
+//    }
+//  }
```
Comment (fboulnois) [inline src/main/java/org/broadinstitute/consent/http/resources/DatasetResource.java]: Does this need to be removed?

**Review by fboulnois — APPROVED** [review]
code cleanup 👍

**Review by okotsopoulos — APPROVED** [review]
Looks good but there are a few changes that look like they'd be reformatted following a linter run, would you mind running that?

More generally, what would you think about putting a lint check in PRs?

**Comment by rushtong** [comment]
> Looks good but there are a few changes that look like they'd be reformatted following a linter run, would you mind running that?
> 
> More generally, what would you think about putting a lint check in PRs?

@okotsopoulos - I created https://broadworkbench.atlassian.net/browse/DCJ-646 for this suggestion, thank you! I'll reformat this PR after rebasing from other changes.

**Pull request merged**